apps/src/code-studio/components/progress/UnitOverviewTopRow.jsx

```diff
@@ -126,8 +126,6 @@ class UnitOverviewTopRow extends React.Component {
       courseVersionId
     } = this.props;
 
-    const useMigratedTeacherResources = isMigrated && !teacherResources.length;
```

Comment:
Do the teacher resources on the scripts need to be cleaned up?

Reply:
yes, the `teacher_resources` a.k.a. unmigrated teacher resources can be removed after this. I'll prep the PR for that now so we can review it while this topic is fresh in our minds.